src/lib/operators/abstract_aggregate_operator.cpp

```diff
@@ -8,10 +8,10 @@
 namespace opossum {
 
 AbstractAggregateOperator::AbstractAggregateOperator(
-    const std::shared_ptr<AbstractOperator>& in, const std::vector<std::shared_ptr<AggregateExpression>>& aggregates,
+    const std::shared_ptr<AbstractOperator>& op, const std::vector<std::shared_ptr<AggregateExpression>>& aggregates,
```

Comment:
Pick here: I know we allowed `op` (and apparently not `in`) to have two characters. However, I'm not glad with many operator constructors having `op` to indicate the *input* operator. So maybe we allow `in` or use `input`/`input_operator`?
Intuitively, I would guess `op` in an operator constructor would be related to some copy constructor.

Reply:
Changed to `input_operator`.

I will look for other occurrences.